exonum-java-binding/core/src/main/java/com/exonum/binding/core/runtime/Pf4jServiceLoader.java

```diff
@@ -131,8 +132,10 @@ private LoadedServiceDefinition loadDefinition(String pluginId) throws ServiceLo
   private static ServiceArtifactId extractServiceId(String pluginId)
       throws ServiceLoadingException {
     try {
-      JavaArtifactNames.checkArtifactName(pluginId);
-      return ServiceArtifactId.newJavaId(pluginId);
+      String[] result = JavaArtifactNames.checkPluginArtifact(pluginId);
```

Comment:
The task description explicitly requests that pluginId is made equal to Exonum artifact id. https://jira.bf.local/browse/ECR-4041